content/en/docs/instrumentation/net/automatic/_index.md

```diff
@@ -267,7 +267,7 @@ For more information, see:
 ## Supported libraries and frameworks
 
 The OpenTelemetry .NET Automatic Instrumentation supports a wide variety of
-libraries. For a complete list, see [Instrumentations](./instrumentations).
+libraries. For a complete list, see [Instrumentations](./instrumentations.md).
```

Comment:
By adding the `.md` the link works in the github repo but not when deployed. This is unfortunately a limitation of hugo&github (or any other markdown viewer)